content/en/monitors/monitor_types/log.md

```diff
@@ -10,88 +10,83 @@ further_reading:
-    
-2. Choose a [Measure][1] or [Facet][2] to monitor. [Measure][1] lets you choose the aggregation function whereas [Facet][2] displays a count.
-    
-    {{< img src="monitors/monitor_types/log/choose_measure_facet.png" alt="choose measure facet" responsive="true" style="width:50%;">}}
-    
-3. Select the aggregation function for the [Measure][1] you want to monitor:
-    
-    {{< img src="monitors/monitor_types/log/agg_function.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
+### Define the search query
 
-4. (Optional) Define the alert grouping:
-  {{< img src="monitors/monitor_types/log/log_monitor_group_by.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
-    With or without alert grouping defined, you get **one** alert when the aggregated value meets the conditions set below. Even if you split the query by host, a single notification is sent if several hosts meet the conditions set below. This is done to reduce notification noise.
+As you define the search query, the graph above the search fields updates.
 
-5. Set alert conditions. The following options can be used:
+1. If you have [multiple log indexes][2], select the index to search.
+2. Construct a search query using the same logic as a [log explorer search][3].
+3. Choose to monitor over a log count, [facet][4], or [measure][5]:
+    * **Monitor over a log count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the amount of logs over a selected time frame then compares it to the threshold conditions.
+    * **Monitor over a facet**: If a [facet][4] is selected, the monitor alerts over the `Unique value count` of the facet.
+    * **Monitor over measure**: If a [measure][5] is selected, the monitor alerts over the numerical value of the log facet (similar to a metric monitor) and aggregation needs to be selected (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
+4. Define the alert grouping (optional). **Note**: With or without alert grouping defined, you get **one** alert when the aggregated value meets the set conditions. Even if you split the query by host, a single notification is sent if several hosts meet the set conditions. This is done to reduce notification noise.
 
-  {{< img src="monitors/monitor_types/log/above_below.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
+{{< img src="monitors/monitor_types/log/define-the-search-query.png" alt="Below monitor for backend service" responsive="true" style="width:60%;" >}}
 
-6. Configure your **notification options**:  
+### Set alert conditions
 
-  {{< img src="monitors/monitor_types/log/set_alert_conditions.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
+* Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
```

Comment:
```suggestion
* The threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
```

Reply:
I've been doing this on purpose because it's the way it is in the UI. What do you think?

Reply:
oh! just realized that this is a sentence continuing on from the first bullet. 👍